scripts/sync_icons.sh

```diff
@@ -230,6 +236,12 @@ __attribute__((visibility("default")))
   return [self pathForIconName:kIconName withBundleName:kBundleName];
 }
 
++ (nullable UIImage *)imageFor_$icon_name {
+  NSBundle *bundle = [self bundleNamed:kBundleName];
+  return [UIImage imageNamed:kIconName inBundle:bundle
+             compatibleWithTraitCollection:nil];
```

Comment:
If you line break in the middle of a method call, all parameters must be on their own line.
Or, to put it another way:
return [UIImage imageNamed:
                                  inBundle:
    compatibleWithTraitCollection:];

Note that compatible won't fit if you try and align colons, so it is indented 4 spaces.